code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -405,4 +405,11 @@
 	desc = "100% synthetic \"Country Girls LLC.\" brand mouth wheat. Warning: not for actual consumption."
 	icon_state = "mouthwheat"
 	w_class = ITEMSIZE_SMALL
+	body_parts_covered = 0
+
+/obj/item/clothing/mask/kitsune

```

Comment:
Please //ChompEDIT START ... //ChompEDIT END block comment code in upstream touched files, or put edits in modular_chomp code files instead